src/commcare_cloud/ansible/roles/kafka/templates/server.properties.j2

```diff
@@ -117,3 +120,9 @@ zookeeper.connect={% for host in groups['zookeeper'] %}{{ host }}:{{ zookeeper_c
 
 # Timeout in ms for connecting to zookeeper
 zookeeper.connection.timeout.ms=1000000
+{% if kafka_version == "2.4.1" %}
+inter.broker.protocol.version=0.8.2.2
+log.message.format.version=0.8.2.2
```

Comment:
what's the plan for changing these to `2.4` once the migration is complete? I still this this needs some work. I'll try take a look today.

Reply:
last we looked at it @rohit-dimagi said he could not find a anyway to do it in Ansible , he suggested the second step to be a manual one